ONSdigital/census-rh-ui
# CR-1253 update translation files

# Motivation and Context
Add welsh translations for the 'Find a support centre' page

# What has changed
Regeneration of translation files
Removed redundant translations
Add new translations for 'Find a support centre' page

# How to test?
Go to /en/find-a-support-centre/
Switch to welsh via the Cymraeg link (top right)
Ensure that text is in Welsh

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1253
